code/modules/uplink/uplink_items/uplink_clothing.dm

```diff
@@ -21,26 +21,15 @@
 	cost = 1
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops) //They already get these
 
-/datum/uplink_item/suits/padding
```

Comment:
Why are you removing all this?

Reply:
_And finally, for balance concerns regarding this thing, the armor accessories have been axed due to essentially being made redundant and armor stacking is very scary_

consider the thought of someone spending five tc (or maybe more might bump up the price) and getting really fucking good armor to the chest

these things were pretty mediocre at best while being incredibly boring, pay two telecrystals for a good bit of armor which nobody can see, also it only works to the chest so good luck if anyone (everyone whos ever played) aims for the head and if i was marginally decent at code id find a way to make the sneaksuit unable to have items attached to it for being too _dummy thick_ but thats out of my paygrade

tldr they suck

Reply:
They covered legs, arms and chest, by the by. They were actively got by people and are not all that bad in a `I want to not die from 2 shot memes`. Imo they should stay, no reason to remove them. People use them People like em

Reply:
okay fine i give now you can buy them but you cant put them on the blood red sneaksuit for the fear of armor stacking i hate coding